### ISSUE TITLE
Unable to Reproduce Results in the paper

### ISSUE THREAD

**Issue by stephen016** [MANGA-UOFA/PTfer#2]
Hello,  the project is very interesting and I am trying to reproduce the results reported in the paper using the code provided in this repository. However, I am encountering difficulties and unable to achieve the same outcomes. The prompt transfer performance is much lower than the number reported in the paper. 

I followed the instructions in the repo, got prompt from OptiPrompt first and ran this repository's code. Could you share the specific hyperparameters used in your experiments or your experiment log? Any additional guidance on faithfully reproducing your results would be much appreciated. Thanks in advance for your help/

**Comment by khalilbalaree** [comment]
Hi, thanks for your interest in our work. I realized that for extracting anchors, I deleted some code in rel2abs_util.py by mistake. 

In our main experiment, we utilize the common vocab from OptiPrompt for anchors, whereas for the alternative method, when common vocab path is not specified, the automatic extraction may lead to low quality anchors. I update the file rel2abs_util.py, which is our original version. This fix should let you replicate our results. Thank you!
